docs/release/trg-5/trg-5-6.md

```diff
@@ -0,0 +1,32 @@
+---
+title: TRG 5.06 - Application Configuration
+---
+
+| Author               | Status | Created     | Post-History    |
+|----------------------|--------|-------------|-----------------|
+| Catena-X System Team | Active | 30-Nov-2022 | Initial release |
+
+## Description
+
+Every configuration aspect of your application has to be configurable for your Helm Chart.
+
+## Why
+
+The application will be deployed in different environments and application operation must be able to adjust the
+deployment to environmental needs.
+
+Examples for configurable items:
+
+- Ingress class, URLs, TLS configuration, etc.
+- Resource annotations and labels
+- Signing keys
+- Database connection strings
+- Secrets (with empty values!)
+- Persistent Storage Claims
```

Comment:
I think its the storage class of a pvc? Its also persistent volumne claim

Reply:
changed wording